acq4/devices/DAQPressureControl/pressurecontrol.py

```diff
@@ -1,10 +1,18 @@
 from __future__ import print_function
-from acq4.util import Qt
+
 from ..PressureControl import PressureControl
 
 
 class DAQPressureControl(PressureControl):
     """Pressure control device driven by DAQ analog/digital channels.
+    The configuration for these devices should look like:
+        sources:
+            regulator:
+                channel_name: TODO what?
```

Comment:
sources:
        regulator:
            valve_1: 1  # activate only valve 1 for regulator
            valve_2: 0
        atmosphere:
            valve_1: 0  # deactivate all valves for atmosphere
            valve_2: 0
        user:
            valve_1: 0  # activate only valve 2 for user
            valve_2: 1